packs/data/feats.db/advanced-witchcraft.json

```diff
@@ -29,7 +29,35 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "adjustName": false,
+                "allowedDrops": {
+                    "label": "PF2E.SpecificRule.Prompt.AdvancedMulticlassFeat",
```

Comment:
Repeating the prompt in the message isn't going to look right. It'll need a distinct phrase.

Reply:
I've updated with a new phrase for both the advanced and basic multiclass feats, but it looks like this is already an issue for every feat that does this.